.stylelintrc.json

```diff
@@ -2,6 +2,7 @@
   "extends": "@brightspace-ui/stylelint-config",
 	"ignoreFiles": [
     "components/demo/code-dark-plus-styles.js",
-    "components/demo/code-tomorrow-styles.js"
+    "components/demo/code-tomorrow-styles.js",
+    "components/skeleton/skeleton-mixin.js"
```

Comment:
Stylelint hates the inline CSS literals on line 9 of this file. There's [another plugin](https://www.npmjs.com/package/postcss-lit) for Lit specifically that we might have better luck with -- I'll try to look into it separate from this fix.